### ISSUE TITLE
Encryption causes Albius to fail during setup

### ISSUE THREAD

**Issue by Aeases** [Vanilla-OS/Albius#47]
Title says it all, with Encryption enabled my install would fail, It may have failed at more steps than just step 20 at times, (step 7?) but I was testing multiple different things, Step 20 was the most consistant, & I have a screenshot of it.

![](https://cdn.discordapp.com/attachments/1023245426385760267/1207887251019669604/IMG_4628.jpg?ex=65e147aa&is=65ced2aa&hm=5e18888c4ff8fcc02442cff1f57da185c35d243ee910d4710fb877d2154a5a61&)

## System Info
![image](https://github.com/Vanilla-OS/Albius/assets/96164686/c22eae5e-c5f4-4237-8d63-f4931cb4921a)

**Comment by Thatoo** [comment]
I face the same issue with `Vanilla OS 2 beta #117`

**Comment by mirkobrombin** [comment]
I am able to install vos 2 with LUKS enabled

**Comment by taukakao** [comment]
If you used manual paritioning, please provide your exact partition layout so we can try to replicate this.

**Comment by Thatoo** [comment]
I didn't make any manual partitioning. I just set a new VM in Virtualbow as I was doing during `alpha` test and for `beta #113`, the VM is new, the disk is "new", I mean it's a .vdi file as usual in Virtualbox, nothing different than previous test but this time, it fails everytime, either because of encryption as shown by @Aeases  but it fails also without encryption as I explain in https://github.com/Vanilla-OS/Albius/issues/48 and https://github.com/Vanilla-OS/Albius/issues/49 .

Basically, I couldn't install the `beta #117` on VirtualBox.

**Comment by Thatoo** [comment]
I've just tried again with the  # 113 version (first beta from the 30/01/2024 and installation with encryption works.

**Comment by taukakao** [comment]
Should be fixed in \#120 if it's not, reopen this issue
